static/app/gettingStartedDocs/javascript/nuxt.tsx

```diff
@@ -0,0 +1,391 @@
+              {tct(
+                'For the server, create a [codeFile:sentry.server.config.ts] file in your project root and initialize the Sentry SDK:',
+                {codeFile: <code />}
+              )}
+
+              <StyledAlert type="info" showIcon>
+                {tct(
+                  'To complete the server-side setup, follow the [link:Sentry Nuxt docs] for guidance. Nuxt compiles your code in ESM on the server side as well, so the deployment setup can get tricky depending on where you deploy your application.',
+                  {
+                    link: (
+                      <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/nuxt/#server-side-setup" />
+                    ),
+                  }
+                )}
+              </StyledAlert>
+            </Fragment>
+          ),
+          code: [
+            {
+              label: 'TypeScript',
+              value: 'typescript',
+              language: 'typescript',
+              code: getSdkServerSetupSnippet(params),
+            },
+          ],
+        },
+      ],
+    },
+    {
+      title: t('Upload Source Maps'),
+      description: tct(
+        'To upload source maps to Sentry, follow the [link:instructions in our documentation].',
+        {
+          link: (
+            <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/nuxt/#add-readable-stack-traces-to-errors" />
+          ),
+        }
+      ),
+    },
+  ],
+  verify: () => [
+    {
+      type: StepType.VERIFY,
+      description: t(
+        "This snippet contains an intentional error and can be used as a test to make sure that everything's working as expected."
+      ),
+      configurations: [
+        {
+          code: [
+            {
+              label: 'JavaScript',
+              value: 'javascript',
+              language: 'html',
+              code: getVerifyNuxtSnippet(),
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  nextSteps: () => [
+    {
+      id: 'nuxt-features',
+      name: t('Nuxt Features'),
+      description: t('Learn about our first class integration with the Nuxt framework.'),
+      link: 'https://docs.sentry.io/platforms/javascript/guides/nuxt/features/',
+    },
+    {
+      id: 'performance-monitoring',
+      name: t('Performance Monitoring'),
+      description: t(
+        'Track down transactions to connect the dots between 10-second page loads and poor-performing API calls or slow database queries.'
+      ),
+      link: 'https://docs.sentry.io/platforms/javascript/guides/nuxt/tracing/',
+    },
+    {
+      id: 'session-replay',
+      name: t('Session Replay'),
+      description: t(
+        'Get to the root cause of an error or latency issue faster by seeing all the technical details related to that issue in one visual replay on your web application.'
+      ),
+      link: 'https://docs.sentry.io/platforms/javascript/guides/nuxt/session-replay/',
+    },
```

Comment:
We have a open [PR](https://github.com/getsentry/sentry/pull/78461/files) that removes the product specific next steps.
```suggestion
```